src/control/controlobject.cpp

```diff
@@ -103,7 +103,7 @@ void ControlObject::setParameterFrom(double v, QObject* pSender) {
 
 // static
 void ControlObject::set(const ConfigKey& key, const double& value) {
-    QSharedPointer<ControlDoublePrivate> pCop = ControlDoublePrivate::getControl(key);
+    auto pCop = ControlDoublePrivate::getControl(key);
```

Comment:
The function name getControl() does not reveal that it returns a `QSharedPointer<ControlDoublePrivate>`  so we need to keep the full type of rename the function.